tutorials/scripting/c_sharp/c_sharp_exports.rst

```diff
@@ -267,6 +267,18 @@ when editing.
     [Export(PropertyHint.ExpEasing)]
     public float TransitionSpeed { get; set; }
 
+Export with Suffix Hint
+-----------------------
+
+Display a unit hint suffix for exported variables. Works with numeric types, such as floats or vectors:
+
+.. code-block:: csharp
+
+    [Export(PropertyHint.None,"suffix:m/s\u00b2")]
```

Comment:
Looking at this again, using `\u00b2` for the squared symbol may merit a comment or a mention in the preceding sentence. Personally I think it's fine as is - curiosity is a good teacher.

Other than that, LGTM but my approval as just a contributor is not the one that matters for getting these changes merged.

Reply:
Hey @tetrapod00 thanks for the feedback!

I agree that it's probably a good idea to add a comment / clarification regarding the use of `\u00b2`, I'll wait for input from maintainers to see how they'd like to proceed.